cmd/lakectl/cmd/store/s3.go

```diff
@@ -27,6 +28,10 @@ type S3Walker struct {
 	s3 s3iface.S3API
 }
 
+func (s *S3Walker) SetS3(s3 s3iface.S3API) {
```

Comment:
Would prefer to expose the member, so I can initial the structure set/get this member, vs just enable a setter func.
But for this test, we don't need to expose a setter.  Calling `store.Walk` function will get us this walker and better code cover.